### PR TITLE
Implemented get setting names, 

### PR DESCRIPTION
**Pull request by Gaspard--**
Untested because I have no time, I need to integrate stuff into feathers.

### CODE REVIEW

File: lib/Config.cpp [hunk 1]
```diff
@@ -45,6 +45,13 @@ void Albinos::Config::parseResponse(json const &data)
     return;
   } catch (...) {
   }
+
+  try {
+    settingNames.reset(new std::vector<std::string>(std::move(data.at("SETTINGS_NAME").get<std::vector<std::string>>())));
```
Comment (Riz-cros) [inline lib/Config.cpp]: le bon field c'est SETTINGS_NAMES il me semble, si tu cherche a recupe le resultat d'un get settings names

Comment (Riz-cros) [reply lib/Config.cpp]:
```suggestion
    settingNames.reset(new std::vector<std::string>(std::move(data.at("SETTINGS_NAMES").get<std::vector<std::string>>())));
```

File: lib/Config.hpp [hunk 2]
```diff
@@ -40,6 +40,7 @@ namespace Albinos
 
     std::optional<KeyWrapper> key;
     std::optional<KeyWrapper> roKey;
+    std::unique_ptr<std::vector<std::string>> settingNames;
```
Comment (Riz-cros) [inline lib/Config.hpp]: would be better to name it `settingsNames` to be consistent but it's not a big deal

File: lib/Config.cpp (hunk 1 above)
Comment (Gaspard--) [reply lib/Config.cpp]: Ah, that's not proper English so I got tricked :P

Comment (Riz-cros) [reply lib/Config.cpp]: its not ?

Comment (Gaspard--) [reply lib/Config.cpp]: I don't know how to explain, but I guess "setting name" is a compound noun and therefor it's pluralised to "setting names".

**Pull request merged**